internal/pkg/pluginengine/change.go

```diff
@@ -169,3 +182,47 @@ func handleResult(smgr statemanager.Manager, change *Change) error {
 	log.Successf("Plugin %s(%s) %s done.", change.Tool.Name, change.Tool.Plugin.Kind, change.ActionName)
 	return nil
 }
+
+// FillInputParams fill inputs from state
+func fillRefValueWithOutputs(smgr statemanager.Manager, options map[string]interface{}) error {
+	// traverse options
```

Comment:
I think this line of comment is redundant. `for ... range ...` is already quite self-explanatory.